AppBidding-MoPub/AppBidding-MoPub/InterstitialViewController.m

```diff
@@ -0,0 +1,65 @@
+//
+//  InterstitialViewController.m
+//  AppBidding-MoPub
+//
+//  Copyright © 2020 Criteo. All rights reserved.
+//
+
+
+#import "InterstitialViewController.h"
+#import "AdConfigurations.h"
+@import CriteoPublisherSdk;
+
+@interface InterstitialViewController ()
+@property (weak, nonatomic) IBOutlet UIButton *displayInterstitialButton;
+
+@end
+
+
+@implementation InterstitialViewController
+
+
+- (void) viewDidLoad {
+    [super viewDidLoad];
+    
+    // Instantiate the interstitial using the class convenience method.
+    self.interstitial = [MPInterstitialAdController interstitialAdControllerForAdUnitId:[AdConfigurations mopubInterstitialAdUnitId]];
+    
+    self.interstitial.delegate = self;
+    [self loadInterstitial];
+}
+
+
+- (void)loadInterstitial {
+    
```

Comment:
this is an unnecessary empty line

Reply:
It is necessary: the method loadInterstitial is called in the viewDidLoad.

Reply:
empty line removed